app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardAzure/WizardAzure.tsx

```diff
@@ -38,7 +40,15 @@ const initialValues: FormValues = {
   },
 };
 
-const options = [buildOption('api', 'svg-api', 'API', '', 'api')];
+const options = [
```

Comment:
nice! just to make sure we don't miss required props. you can type the array
```suggestion
const options: Array<BoxSelectorItem<'api'>> = [
```

Reply:
updated